checkov/common/runners/object_runner.py

```diff
@@ -92,8 +92,10 @@ def run(
                     continue
 
                 # result record
-                if result_config:
+                if result_config and isinstance(result_config, (list, dict)):
                     end, start = self.get_start_end_lines(end, result_config, start)
+                else:
+                    continue
```

Comment:
so in this case we will not generate record?

Reply:
@PelegLi maybe it will be better to try catch here instead of checking this specific case